tools/vep_somatic_annotate_r93.cwl

```diff
@@ -0,0 +1,102 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: kfdrc-vep-somatic-annotate
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 24000
+    coresMin: 16
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/vep:r93.7'
+baseCommand: [mkdir]
+arguments:
+  - position: 1
+    shellQuote: false
+    valueFrom: >-
+      $(inputs.species)
+      && tar --use-compress-program="pigz -p 8" -xf $(inputs.cache.path) -C $(inputs.species)
+      && perl /ensembl-vep-release-93.7/vep
+      --af
+      --af_1kg
+      --af_esp
+      --af_gnomad
+      --allele_number
+      --assembly $(inputs.ref_build)
+      --biotype
+      --buffer_size 10000
+      --cache
+      --cache_version $(inputs.cache_version)
+      --canonical
+      --ccds
+      --check_existing
+      --dir_cache $(inputs.species)
+      --domains
+      --failed 1
+      --fasta $(inputs.reference.path)
+      --flag_pick_allele
+      --fork 16
+      --format vcf
+      --gene_phenotype
+      --hgvs
+      --input_file $(inputs.input_vcf.path)
+      --no_escape
+      --no_progress
+      --no_stats
+      ${
+        if(inputs.merged){
+          return "--merged";
+        }
+        else{
+          return "";
+        }
+      }
+      --numbers
+      --offline
+      --output_file $(inputs.output_basename).$(inputs.tool_name).PASS.vep.vcf
+      --pick_order canonical,tsl,biotype,rank,ccds,length
+      --polyphen b
+      --protein
+      --pubmed
+      ${
+        if (!inputs.species || inputs.species != "canis_familiaris"){
```

Comment:
I think I'd prefer to have this check specifically for if the species is Homo sapiens and / or some variants on that (human, H. sap) and then adding the `--regulatory` option instead of only checking if `inputs.species` is blank or one specific species. If the intent is to only allow human or dog to run, maybe lock the option down to one of only two values. Is the species hard coded somewhere? Dogs species name should be Canis lupus familiaris.

Reply:
So, the species is hard coded into the VEP cache. Your comment prompted me to check what the dog one is hard-coded as, and in seems in VEP93, it's `canis_familiaris`, but in the newer caches it is `canis_lupis_familiaris`. Perhaps I should make including this a boolean instead and explain that there are times when this option is not available in the cache. Thanks for the input!

Reply:
Added another VEP test run based on this comment:
https://cavatica.sbgenomics.com/u/zhangb1/kf-somatic-tools-test/tasks/d100da6b-52b2-4f16-a3a1-4aa26aa5062f